package.json

```diff
@@ -44,7 +46,7 @@
     "@typescript-eslint/eslint-plugin": "5.37.0",
     "@typescript-eslint/parser": "5.37.0",
     "concurrently": "7.4.0",
-    "eslint": "8.23.1",
+    "eslint": "8.22.0",
```

Comment:
That's for WebStorm to work for me...are you okay with that small downgrade for now? :c